lib/Dialect/AIE/IR/AIEDialect.cpp

```diff
@@ -1137,6 +1158,27 @@ int SwitchboxOp::getNumDestConnections(WireBundle bundle) {
     default:
       return 0;
     }
+  else if (getTileOp().isMemTile())
+    switch (bundle) {
+    //case WireBundle::Core:
```

Comment:
What is the purpose of the commented lines here and in getNumSourceConnections ?

Reply:
Matching the style of prior code. Will remove.